service/security/shiro/src/main/java/org/eclipse/kapua/service/authentication/credential/shiro/CredentialImpl.java

```diff
@@ -171,7 +171,7 @@ public void setUserId(KapuaEid userId) {
     }
 
     public CredentialStatus getCredentialStatus() {
```

Comment:
Remove this method because it is a duplicate of `getStatus()`.
Fix any reference to this method then.

Reply:
Ok, I will close this pull request and recommit. Thanks.

Reply:
@miricy thank you for the fixes. 

Next time please continue commit on the same branch, so all changes to this PR will be tracked on a single place. It's easier to then keep track of the changes. Now I'll have a look at the other PR you opened.